tools/tune/src/src_export_coef.m

```diff
@@ -46,6 +46,7 @@
 	fprintf(fh, ' *\n');
 	fprintf(fh, ' */\n');
 	fprintf(fh, '\n');
+	fprintf(fh, '/** \\cond GENERATED_BY_TOOLS_TUNE_SRC */\n');
```

Comment:
```suggestion
	fprintf(fh, '%s\n', /** \cond GENERATED_BY_TOOLS_TUNE_SRC */');
```

Reply:
Your suggestion works, you can run the line from command prompt like this:

```
>> fprintf(1, '%s\n', '/** \cond GENERATED_BY_TOOLS_TUNE_SRC */');
/** \cond GENERATED_BY_TOOLS_TUNE_SRC */
```

But I prefer the simpler \ to escape the backslash character. Your way feels like a hazard, doesn't it?

Reply:
> Your way feels like a hazard, doesn't it?

? I think confusing the format string and the message is the hazard actually, it's just one step away from `warning: format not a string literal`. I think it's "Good Practice" to always keep them separate and never think about it.

Doesn't matter here, it's not like you can write memory corruption bugs in matlab...